src/users_db.rs

```diff
@@ -264,7 +285,14 @@ impl UsersDb {
                 email       TEXT NOT NULL UNIQUE,
                 password    TEXT NOT NULL,
                 secret      TEXT NOT NULL,
-                is_admin    BOOL NOT NULL DEFAULT 0
+                is_admin    BOOL NOT NULL DEFAULT 0,
+                push_uri    TEXT,
+                push_key    TEXT
```

Comment:
The token already contains the user identifier inside it. It's a signed JWT that you can decode to obtain that value.

Check https://github.com/fxbox/users/blob/master/src/auth_middleware.rs#L43

Since the token format will likely change once we have authorization, we could add a function to this module to get the user identifier from a given token.